doc/advanced_configuration.rst

```diff
@@ -397,6 +397,42 @@ The default behavior is ``sphinx_gallery_thumbnail_number = 1``. See
 :ref:`sphx_glr_auto_examples_plot_choose_thumbnail.py` for an example
 of this functionality.
 
+.. _binder_links:
+
+Generate Binder links for gallery notebooks
+===========================================
+
+Sphinx-Gallery automatically generates Jupyter notebooks for any
+examples built with the gallery. `Binder <http://mybinder.org>`_ makes it
+possible to create interactive GitHub repositories connect to cloud resources.
+If you host your documentation on a GitHub repository, it is possible to
+auto-generate a Binder link for each notebook. Clicking this link will
+take users to a live version of the Jupyter notebook where they may
+run the code interactively. For more information see the `Binder documentation
+<http://docs.mybinder.org>`_.
+
+.. warning::
+
+   Binder is still beta technology, so there may be instability in the
+   experience of users who click Binder links.
+
+In order to enable Binder links with Sphinx-Gallery, you must specify
+a few pieces of information in ``conf.py``. These are given as a nested
+dictionary following the pattern below::
+
+    sphinx_gallery_conf = {
+      ...
+      'binder': {
+         'org': '<github_org>',
+         'repo': '<github_repo>',
+         'url': '<binder_url>',  # URL serving binders (e.g. mybinder.org)
+         'branch': '<repo_branch>',  # Can also be a tag or commit hash
+         'dependencies': '<list_of_paths_to_dependency_files>'  # Dependency files Binder will use to build the environment. See docs.mybinder.org for details.
```

Comment:
instead of a long code comment, better to add a note in the sentences below with a proper link